src/systems/environment_preload/EnvironmentPreload.cc

```diff
@@ -93,32 +93,52 @@ void EnvironmentPreload::PreUpdate(
       return;
```

Comment:
@arjo129 if we add options to `EnvironmentPreload` plugin, we should probably replicate for the `EnvironmentLoader` GUI plugin.